src/components/DatatableV2/Table/components/TableCheckbox.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { forwardRef } from 'react';
+import cls from 'classnames';
+
+import * as checked from '../../../../theme/icons/check';
+import * as indeterminate from '../../../../theme/icons/minus';
+import { Checkbox } from '../../../forms';
+import { TableCheckboxProps } from './TableCheckbox.types';
+
+const generateIconProps = ({ width, height, svgPathData }) => ({
```

Comment:
duplicate in `forms/CheckBox`

Reply:
that's correct, we can't use that one from `forms` due to performance of styled components

Reply:
I meant `generateIconProps()` is an exact copy

Reply:
could be moved to `utils/helpers`

Reply:
Oh I get it, currently, I have no plan on how to progress with Checkbox so I didn't want to create any dependency outside of the Datatable component.